app/src/main/java/com/devshish/internship/presentation/ui/home/HomeFragment.kt

```diff
@@ -1,6 +1,48 @@
 package com.devshish.internship.presentation.ui.home
 
+import android.os.Bundle
+import android.view.View
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
 import com.devshish.internship.R
+import com.devshish.internship.databinding.FragmentHomeBinding
+import com.devshish.internship.presentation.ui.home.artist.ArtistItemAdapter
+import com.devshish.internship.presentation.ui.home.tracks.TrackItemAdapter
+import com.zhuinden.fragmentviewbindingdelegatekt.viewBinding
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class HomeFragment : Fragment(R.layout.fragment_home) {
+
+    private val binding by viewBinding(FragmentHomeBinding::bind)
+    private val viewModel: HomeViewModel by viewModel()
+
+    private val artistItemAdapter = ArtistItemAdapter()
+    private val trackItemAdapter = TrackItemAdapter()
```

Comment:
Maybe use here concatAdapter? And you can move title into adapter too so you will have only one recycler view